### ISSUE TITLE
jpype._jexception.RuntimeExceptionPyRaisable

### ISSUE THREAD

**Issue by SumitrajPatil** [FraBle/python-duckling#39]
Hi I am trying to run python wrapper but I'm getting `  raise _RUNTIMEEXCEPTION.PYEXC("Class %s not found" % name)
jpype._jexception.RuntimeExceptionPyRaisable: java.lang.RuntimeException: Class clojure.java.api.Clojure not found`

I'm running simple example

` from duckling import Duckling
d = Duckling()
d.load()
print(d.parse('tomorrow'))`

I'm not sure what I'm doing wrong, may be in installations. I have installed
 Duckling 1.8
JPype1 0.6.3
And I'm running it ancaonda env.

**Comment by SumitrajPatil** [comment]
I have got the solution Thanks.

**Comment by mekyush** [comment]
@SumitrajPatil  I am also getting same error. What was the resolution?

**Comment by SumitrajPatil** [comment]
> @SumitrajPatil I am also getting same error. What was the resolution?

Hey, I found the solution but I don't remember exactly what was the solution. Sorry for not responding with the solution earlier.

**Comment by sumitovant-alan-menaged** [comment]
I ran into this issue as well and found the solution. I had been importing another python library that already initialized a JVM. So when I tried to initialize `Duckling()`, it skipped over the JVM creation and also skipped over the additions to the classpath. I was able to fix it by removing the other library which creates a JVM. But I would guess there is also a way to use the `jvm_started=True` flag and load the required jars into your existing JVM manually. Hope this helps!
